scripts/helpers/downloads.js

```diff
@@ -139,11 +139,19 @@ function resolveUrl (item, version) {
 
 module.exports = (version) => {
   let downloads = latestDownloads
-  if (semver.satisfies(version, '< 8.0.0')) {
-    downloads = postMergeDownloads
-  }
   if (semver.satisfies(version, '< 1.0.0')) {
     downloads = legacyDownloads
+  } else if (semver.satisfies(version, '< 8.0.0')) {
+    downloads = postMergeDownloads
+  } else if (semver.satisfies(version, '> 10.0.0')) {
```

Comment:
should this be `>= 10.0.0` or is it unrelated cause this will never run again for 10.0.0 ?

Reply:
So are you sure that from the ref #1769, it should be >= 10.0.0? I'm not sure. I see it says `Node 10+`……?